SemiTruck.h

```diff
@@ -1,2 +1,15 @@
 #pragma once
+#include "Vehicle.h"
+
+
+struct SemiTruck : public Vehicle
+{
+	SemiTruck(const std::string& k);
+    virtual ~SemiTruck();
+    SemiTruck(const SemiTruck&);
+    SemiTruck& operator=(const SemiTruck&);
+	void pullOver();
+	void laneChange();
+
```

Comment:
struct is public by default
remove this keyword
Fix this indentation
- [ ] revise